snuba/query/processors/failure_rate_processor.py

```diff
@@ -33,7 +32,11 @@ def process_functions(exp: Expression) -> Expression:
                         binary_condition(
                             None,
                             ConditionFunctions.NOT_IN,
-                            Column(None, None, "transaction_status"),
+                            FunctionCall(
```

Comment:
Where in your code do you decide whether this should be the column or NULL? Because this will default to NULL regardless of which table is being used.

Reply:
fixed